backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/objcexport/ObjCExportHeaderGenerator.kt

```diff
@@ -48,6 +50,11 @@ internal class ObjCExportTranslatorImpl(
 
     private val kotlinAnyName = namer.kotlinAnyName
 
+    //Flag to enable generic output
+    private val writeGenerics:Boolean by lazy {
+        System.getProperty("framework.writeGenerics", "false").toBoolean()
```

Comment:
Maybe instead use property in .def file or CLI argument? We usually do not use Java properties to configure tools behavior.

Reply:
Moved to CLI.